packages/beacon-node/src/node/utils/lightclient.ts

```diff
@@ -0,0 +1,7 @@
+import {LightClientServer} from "../../chain/lightClient/index.js";
+
+export function assertLightClientServer(server: LightClientServer | undefined): asserts server is LightClientServer {
+  if (!server) {
+    throw Error("LightclientServer is not running on this node. To server this endpoint we need this");
```

Comment:
This is an error message we return on the api, can make it more succinct

```suggestion
    throw Error("Light client server is disabled");
```